Readme.md

```diff
@@ -56,40 +56,58 @@ This is very important not doing it either leads to compiler errors or even wors
 
 ```c
 /* init gui state */
-struct nk_context ctx;
-nk_init_fixed(&ctx, calloc(1, MAX_MEMORY), MAX_MEMORY, &font);
+struct nk_context *ctx = (nk_context*) malloc(sizeof(nk_context));
```

Comment:
Nitpick: why on heap? It's a small struct and the less allocated memory you have to manage, the better (and more readable).

Reply:
True, I'll fix this, it's because on my own project there's too much on the stack already.